python/ray/data/read_api.py

```diff
@@ -540,6 +541,67 @@ def read_mongo(
     )
 
 
+@PublicAPI(stability="alpha")
+def read_bigquery(
+    project_id: str,
+    dataset: Optional[str] = None,
+    query: Optional[str] = None,
+    *,
+    parallelism: int = -1,
+    ray_remote_args: Dict[str, Any] = None,
+    **bigquery_args,
+) -> Dataset:
+    """Create an Arrow dataset from BigQuery.
+
+    The data to read from is specified via the ``project_id``, ``dataset``
+    and/or ``query``parameters. The dataset is created from the results of
+    executing``query`` if a query is provided. Otherwise, the entire
+    ``dataset`` will be read.
```

Comment:
```suggestion
    The data to read from is specified via the ``project_id``, ``dataset``
    and/or ``query``parameters. The dataset is created from the results of
    executing``query`` if a query is provided. Otherwise, the entire
    ``dataset`` is read.
```

Reply:
Done